src/components/Header.js

```diff
@@ -1,23 +1,26 @@
 import React from "react";
-import "../Header.css";
+
 
 const Header = () => {
   return (
-    <div>
-      <nav>
-        <div className="logo">
+      <header> 
           <img
             className="imagelogo"
             src="https://logo.clearbit.com/spotify.com"
-            alt="logo"
+            alt=""
           />
-        </div>
-        <div className="name">
-          <h2>Name</h2>
-        </div>
-      </nav>
-    </div>
+          <span>Name</span>
+          <button className="imageDownload">
```

Comment:
To implement the `DownloadButton.js` component, you write it like `<DownloadButton />` and at the top of the file import `DownloadButton.js` from `'./DownloadButton.js'`.

As described in the issue, this component should only be imported directly in the header and only the CSS styles which are currently inline styles should be moved into the `index.css` sheet.

Reply:
Yes, reading back I misunderstood the instruction on that item to 'move' the whole thing whereas you just needed the button adding instead.
I've not much experience with PR's and not had to resolve/correct any issues before. 
So I'm not sure how to go about making the changes i.e. do I have to alter it here or do another PR after correcting it in VSCode etc.? If you know a good resource that explains this then would be good to know and I can go learn what is required. Cheers

Reply:
You can make changes directly in your current branch and push it. The PR is then automatically updated. No need for another PR 😊 

If the merge conflicts are still there after syncing, no worries, I will fix them afterwards. 😎

Reply:
I have put back the DownloadButton.js file and added this component to the header.js hopefully as you were expecting.

I then added, committed and pushed it back to my branch and sync'd the branch. However it stated it would remove 2 commits and so I'm not sure if it works? I came to check and noticed it closed the PR automatically so not sure if I did it correctly.

Reply:
If thats works could you please add the hacktoberfest-accepted label so it adds to my tally. Thanks